back/boxtribute_server/graph_ql/definitions/public/queries.graphql

```diff
@@ -3,4 +3,5 @@ type Query {
 
   standardProducts: [StandardProductInfo!]!
   newlyRegisteredBeneficiaryNumbers: BeneficiaryNumbers
+  newlyCreatedBoxes: BoxNumbers
```

Comment:
Can you name the query `newlyCreatedBoxNumbers`? 
When reading `newlyCreatedBoxes`, I'd assume the query returns box objects.